merkato/merkato_config.py

```diff
@@ -3,43 +3,35 @@
 import json
 import os.path
 from merkato.utils import write_to_file, update_config_with_credentials, get_exchange, get_config_selection
+from merkato.utils.database_utils import get_exchange,insert_exchange, no_merkatos_table_exists, create_exchanges_table
 
 def load_config():
     # Loads an existing configuration file
     # Returns a dictionary
-
-    while True:
-        filename = input("Config file name? ")
-
-        if not os.path.isfile("./"+filename):
-            print("File doesn't exist.")
-            continue
-
-        with open("./"+filename) as configuration:
-            data = json.load(configuration)
-
-        # TODO: Error handling and config validation
-
-        return data
-
+    exchange_name = input("what is the exchange name? ")
+    return get_exchange(exchange_name)
+    # TODO: Error handling and config validation
+
+def insert_config_into_exchanges(config)
+    limit_only = config["limit_only"]
+    public_key = config["public_key"]
+    private_key = config["private_key"]
+    exchange = config["exchange"]
+    if no_merkatos_table_exists():
```

Comment:
Why is this checking whether merkatos exists, then creating the exchange table? Shouldnt it be creating the merkatos table? (Or checking if the exchanges table exists)

Reply:
I'm assuming the if no merkatos table check should instead be a check against the exchanges table, makes sense in context

Reply:
yup typo